stable/fairwinds-insights/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "10.0.0"
+appVersion: "10.0.1"
```

Comment:
Not sure which version to use here... Wait for the release that contains the job?